### ISSUE TITLE
[Volunteer Application form content changes]

### ISSUE THREAD

**Issue by Srividya1394** [dev-launchers/dev-launchers-platform#1349]
- Include following sub-text below time commitment field
How many hours a week would you like to volunteer? *
"Note: this role requires at least 10 hours a week."

- Include following sub-text below portfolio/resume field
Please include your portfolio/resume:
Max file size 25MB. Only .doc, .pdf, .png, and .jpg allowed.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBazhXIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--a8dd2fec1899d48b569db48c4762bf452f04170b/image.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbEFXIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--8abb9614743450b228bdde427091b5964aeb4757/image.png)

**Comment by Srividya1394** [comment]
<img width="1360" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL3grQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b3510e78510b8933245574b5ae7efa1e5efca373/image.png" alt="image.png" />

Pre-populated values needs to be removed

**Comment by jeyanthi-sm** [comment]
To keep the time commitment as a single value rather than a range, revert it to the previous state so that the user can use a slider to set the value as it was before.

**Comment by Srividya1394** [comment]
Yes if the previous version has the slider, let us stick to that!

**Comment by chungthuang** [comment]
Time commitment field was added:
<img width="704" height="196" alt="Image" src="https://github.com/user-attachments/assets/15246154-a773-474a-b329-05a246c54f25" />
Max file size was added:
<img width="343" height="111" alt="Image" src="https://github.com/user-attachments/assets/bef9fcbc-31fa-4165-98f3-956d8d890d09" />
